src/utils/camelCase.js

```diff
@@ -7,5 +7,5 @@ export default type =>
     ? camelCase(type)
     : type
         .split(namespacer)
-        .map(camelCase)
+        .map(part => camelCase(part))
```

Comment:
due to `camelCase` accepting multiple arguments, and `map` providing the index as a second argument, and entire array as 3rd.

Reply:
Sure about this? We had it like this before as well with `camelCase` from `lodash`.